bip-wallet-policies.mediawiki

```diff
@@ -298,6 +301,21 @@ Miniscript: A singlesig wallet with automatic inheritance to a timelocked 2-of-3
 
 TBD: add examples with taproot scripts and miniscript.
 
+=== Invalid policies ===
+
+The following descriptor templates are invalid:
+
+* <tt>pkh(@0)</tt>: Key placeholder with no path following it
+* <tt>pkh(@0/0/*)</tt>: Key placeholder with an explicit path present
```

Comment:
This needs to change to `**` from `*` to be a valid case for the non-extended BIP (my bad, sorry).

Reply:
Right! Fixed in 6f98026a2fd27069446fdfcabdfbaad6450b3e23.